GliaWidgets/Coordinator/Engagement/Chat/ChatCoordinator.swift

```diff
@@ -114,7 +104,12 @@ class ChatCoordinator: SubFlowCoordinator, FlowCoordinator {
                 self?.presentWebViewController(with: url)
             }
         }
-        return ChatViewController(viewModel: viewModel, viewFactory: viewFactory)
+
+        let viewController = ChatViewController(viewModel: viewModel, viewFactory: viewFactory)
+
```

Comment:
I suppose newlines here aren't necessary.

Reply:
Fixed in 77b0821a9972ef725218919feaf740a2d35434f6